daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf/daml_lf.proto

```diff
@@ -5,10 +5,10 @@ syntax = "proto3";
 package daml_lf;
 
 option java_package = "com.digitalasset.daml_lf";
-option csharp_namespace = "Com.DigitalAsset.Daml_lf.DamlLf";
+option csharp_namespace = "Com.DigitalAsset.Daml_lf.DamlLfDev";
```

Comment:
This seems weird, why do we only change this for `csharp` but not for `java`?

If that is really intended it needs an entry in the release notes.

Reply:
My mistake. This is for the next PR.

Reply:
fixed